Diagnostics/HealthChecker/DataCollection/ServerInformation/Get-AllNicInformation.ps1

```diff
@@ -240,6 +242,56 @@ Function Get-AllNicInformation {
                                 break
                             }
                         }
+
+                        $ipv4Gateway = @()
+                        $ipv6Gateway = @()
+                        foreach ($gateway in $adapterConfiguration.DefaultIPGateway) {
```

Comment:
Is it really possible to have multiple DefaultIPGateways for both ipv6 and ipv4 on the same nic?

Reply:
Yes, I think it was possible. Will verify.

Reply:
Okay, well the new code that I added should handle this.

Reply:
Validated in lab. We're good to merge this change into main.